crates/full-node/sov-sequencer/src/preferred/rate_limiter/mod.rs

```diff
@@ -91,13 +91,16 @@ const TTL_MULTIPLIER: u64 = 20;
 
 fn clculate_limits<S: Spec>(
```

Comment:
Not changed in this PR but should be a quick fix
```suggestion
fn calculate_limits<S: Spec>(
```

Reply:
fixed